src/backend/joanie/tests/core/test_api_admin_certificate_definitions.py

```diff
@@ -250,3 +250,28 @@ def test_admin_api_certificate_definition_delete(self):
         )
 
         self.assertEqual(response.status_code, 204)
+
+    def test_admin_api_certificate_definition_bulk_delete(self):
+        """
+        Staff user should be able to delete multiple certificate_definitions.
+        """
```

Comment:
Can you write a test to check what append when a resource cannot be deleted.

Reply:
I did that for the models that have a condition where they cannot be deleted :+1: